OptimizelySwiftSDK.podspec

```
@@ -2,12 +2,13 @@ Pod::Spec.new do |s|
   s.name                    = "OptimizelySwiftSDK"
   s.module_name	            = "Optimizely"
   s.version                 = "3.3.1"
-  s.summary                 = "Optimizely experiment framework for iOS/tvOS"
+  s.summary                 = "Optimizely experiment framework for iOS/tvOS/macOS"
```

Can you remove "macOS" from this line? You can use for mac but officially not ready to claim macOS support.
```suggestion
  s.summary                 = "Optimizely experiment framework for iOS/tvOS"
```